docs/concepts/json.md

```diff
@@ -60,6 +60,62 @@ with one noticeable enhancement being that `jiter` supports deserialization of `
 In the future, `jiter` is intended to enable support validation errors to include the location
 in the original JSON input which contained the invalid value.
 
+### Partial JSON Parsing
+
+Starting in v2.7.0 and above, Pydantic's [JSON parser](https://docs.rs/jiter/latest/jiter/) offers support for partial JSON parsing, which is exposed via [`pydantic_core.from_json`][pydantic_core.from_json]. Here's an example of this feature in action:
```

Comment:
Might be worth making this into an admonition.

`<pedant>`"Starting in v2.7.0 and above" is tautologous, I think it should either be "Starting in v2.7.0" or "In v2.7.0 and above"`</pedant>`